### DISCUSSION TITLE
ESLint fixer support

### DISCUSSION THREAD

**Discussion by JDEast1029** [wxmlfile/eslint-plugin-wxml#30] [Ideas]
后面会提供自动修复功能吗？

**Comment by iChenLei** [comment]
会！请问你有在项目使用这个eslint插件吗？除了不支持fixer还有没有别的使用上的问题？ @JDEast1029

**Comment by JDEast1029** [comment]
显示错误的范围还不够准确，整个标签都标红了
![image](https://user-images.githubusercontent.com/16700768/144168869-6e98694e-8eb4-43b8-a752-6c7167e80a0a.png)
